### ISSUE TITLE
Full Page Cache: dynamic hooks not working

### ISSUE THREAD

**Issue by eschiendorfer** [thirtybees/thirtybees#382]
I have a module with a hook "displayDeliveryDate". I marked it green in the performance tab. But the hook is still loading the cached value. When disabling the full page cache, it shows the right value.

Version: 1.0.3, PHP 7.0.24, MySQL 10.2.8-MariaDB

**Comment by firstred** [comment]
Is that the default delivery date module?

**Comment by eschiendorfer** [comment]
No, it's my own module. But a very simple one.

**Comment by firstred** [comment]
A few fixes for the page cache:

https://github.com/thirtybees/thirtybees/commit/7a42de243e60122c0558395bdbe71f55b67b1eb5

**Comment by eschiendorfer** [comment]
@firstred Sorry Michael. I have tried to solve this issue now. Since it some months ago, I first tried to recreate the issue. But I don't manage to... Probably I messed something up, that time. No idea what, since I didn't change my module since then. I will follow this for the next days. If it works as supposed I will close this issue. What should be clear for now: Don't waste any time on this ;)
